server/liveblog/items/items.py

```diff
@@ -50,6 +50,16 @@ class ItemsResource(ArchiveResource):
         'item_type': {
             'type': 'string'
         },
+        'name': {
+            'type': 'string',
+            'minlength': 1,
+            'maxlength': 30
+        },
+        'content': {
```

Comment:
So it's not good for consistency :) We can't have two different fields for the same purpose in _Items_. You should do your constraints verification at the comment level. Or to create a sub model for comments based on _Item_